hub/api/dataset.py

```diff
@@ -233,20 +258,38 @@ def empty(
 
         feature_report_path(path, "empty", {"Overwrite": overwrite})
 
-        storage, cache_chain = get_storage_and_cache_chain(
-            path=path,
-            read_only=False,
-            creds=creds,
-            token=token,
-            memory_cache_size=memory_cache_size,
-            local_cache_size=local_cache_size,
-        )
+        try:
+            storage, cache_chain = get_storage_and_cache_chain(
+                path=path,
+                read_only=False,
+                creds=creds,
+                token=token,
+                memory_cache_size=memory_cache_size,
+                local_cache_size=local_cache_size,
+            )
+        except Exception as e:
+            if isinstance(e, UserNotLoggedInException):
+                message = (
+                    f"Please log in through the CLI in order to create this dataset, "
```

Comment:
Keep the string outside of the class if it is being reused, otherwise modifying multiple duplicate strings becomes a problem at times.